cpu/stm32_common/periph/pm.c

```diff
@@ -29,6 +29,8 @@
 #include "stmclk.h"
 #endif
 
+extern void clk_init(void);
```

Comment:
As long as I can see, this function doesn't exist anymore.

Reply:
It does for L-series CPUs, although defined as static. For F-series, clock initialization was moved to stmclk

Reply:
Yep, forgot to delete